docs/ga4gh-visas.md

```diff
@@ -51,3 +50,20 @@ If an applicant has `researcher-status-by` with value `"so"` or
 checkbox in the applicant details.
 
 See also Bona Fide bot in [bots.md](bots.md).
+
+### European Genome-Phenome Archive (EGA)
+
+The GA4GH Visas are used also by the entitlement push to [EGA](https://ega-archive.org/).
+
+The normal flow is such that:
+- The entitlement push is configured to the REMS instance configuration. See [configuration.md](configuration.md) for the necessary configuration. NB: Remember to enable the EGA support also by `:enable-ega true`. This is done only once.
+- The handlers must each link their ELIXIR user accounts to an EGA user account (an external process).
```

Comment:
Should it be mentioned, that the handlers must log in to REMS using ELIXIR id for all EGA approvals?

Reply:
Yeah makes sense. It is mentioned in the ADR at least. The wording I used in the following bullets hints at it but it could be said explicitly.